src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -493,7 +493,7 @@ protected function moveEntity($id, Request $request)
     }
 
     /**
-     * @param $id
+     * @param int $id
```

Comment:
this is called with `null` for new entities:
```suggestion
     * @param int|null $id
```